src/main/kotlin/com/getkeepsafe/dexcount/OutputFormat.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015-2016 KeepSafe Software
```

Comment:
Yeah, I'd prefer if we kept it intact.  It seems likely to me that these Kotlin files would be construed as derivative works, so best to keep it safe (no pun intended).